### DISCUSSION TITLE
Validation documentation missing "Global Options" code

### DISCUSSION THREAD

**Discussion by NeoCoderMatrix86** [Megabit/Blazorise#4401] [Q&A] [answered]
Hello,

I'm currently migrating much self validated code to the validation component and would like to use a general localizer for validation. The documentation for this part is missing here: "https://blazorise.com/docs/components/validation"

Does anybody have a clue on how to start with a general localizer for validation?

**Comment by stsrki** [comment, marked as answer]
Can you give us a clue about what is a general localizer?

**Reply by NeoCoderMatrix86** [reply]
Well, if i understand the chapter "Global Options" correct, it should be possible to not having to set the localizer for validation each time, but set it once in a startup?!

**Reply by stsrki** [reply]
The example for the global localizer is missing from the docs. We should fix that.

The example should be something like this:

```cs
services
    .AddBlazorise( options =>
    {
        options.ValidationMessageLocalizer = ( message, arguments ) =>
        {
            return "some localized string";
        };
    } 
```)
